source/sync/data-access-patterns/permissions.txt

```diff
@@ -360,6 +353,183 @@ change until the next session.
      write rules. Functions would not operate on a per-document basis.
    - You cannot store permissions information (such as "which document IDs may this user access?") in the user object.
 
+
+.. _flexible-sync-roles-json-reference:
+
+Flexible Sync Roles JSON Reference
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A **Role** object can specify a specific collection's role or the default role
+used for any unspecified collection. You define field-level permissions with a
+**Fields** object in the Role.
+
+.. example::
+
+   The following JSON object describes a **Role** for a team admin. When granted this role, the user:
+
+   - May access any document for reading.
+   - May access a document for writing if the document's ``teamId`` field equals
```

Comment:
I know it's mentioned under the "write" Notes, but wondering if it would be worth reiterating that ``teamId`` has to be a queryable field here